colour/models/rgb/dataset/f_gamut.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+"""
+F-Gamut Colourspace
+===================
+
+Defines the *F-Gamut* colourspace:
+
+-   :attr:`colour.models.F_GAMUT_COLOURSPACE`.
+
+See Also
+--------
+`RGB Colourspaces Jupyter Notebook
+<http://nbviewer.jupyter.org/github/colour-science/colour-notebooks/\
+blob/master/notebooks/models/rgb.ipynb>`_
+
+References
+----------
+-   :cite:`FujiVer1` : F-Log Data Sheet Ver.1.0.
```

Comment:
I can change it but if you want the citation which depends on the entry I added to Mendeley (and will push to the *BIBLIOGRAPHY.bib* file) should be like that:

> -   :cite:`Fujifilm2016` : Fujifilm. (2016). F-Log Data Sheet Ver.1.0. Retrieved from https://www.fujifilm.com/support/digital_cameras/software/lut/pdf/F-Log_DataSheet_E_Ver.1.0.pdf